gen-apidocs/config/v1_18/config.yaml

```diff
@@ -0,0 +1,277 @@
+example_location: "examples"
+api_groups:
+  - "AdmissionRegistration"
+  - "ApiExtensions"
+  - "ApiRegistration"
+  - "Apps"
+  - "AuditRegistration"
+  - "Authentication"
+  - "Authorization"
+  - "Autoscaling"
+  - "Batch"
+  - "Certificates"
+  - "Coordination"
+  - "Core"
+  - "Discovery"
+  - "Extensions"
```

Comment:
Looks to me that this group is dropped in 1.18?

Reply:
OK. Do you think it is a good idea to add the initial files for the release and then iterate on changes?

Reply:
of course!